source/security/index.html.md.erb

```diff
@@ -109,7 +109,7 @@ Your requirements depend on the number of transactions that you process as a
 merchant per scheme (Visa, MasterCard) per year.
 
 For example, if you process 4
-million transactions with Visa and 3 million with MasterCard, the “Fewer than
+million transactions with Visa and 3 million with MasterCard, the "Fewer than
```

Comment:
This I think can remain curly since it’s not code (but possibly single curly quotes because it feels more like a name of something than a direct quotation).